docs/pages/identity-governance/access-request-plugins/ssh-approval-msteams.mdx

```diff
@@ -201,6 +205,9 @@ This should result in a config file like the one below:
 (!examples/resources/plugins/teleport-msteams.toml!)
 ```
 
+The `teams_app_id` is a UUID you can specify that will be used as an identifier
```

Comment:
Is it optional? "You can specify" makes it sound optional.

Reply:
It is required, have changed 'can' to 'must'

Reply:
Looks better, but I'm still confused.

> The `teams_app_id` is a UUID you must specify that will be used as an identifier for the app's manifest within Teams.

Does this mean you can pick any UUID to enter there? If that's the case, why do we require the user to enter one at all - couldn't we just generate it?

Reply:
Looked through the msteams plugin and cant think of a reason we don't here. We could completely remove this field unless there is a benefit to being able to specify an id I can't think of. At the very least we could make it optional and generate it if one isn't provided.

Reply:
Are you sure this doesn't have to match the ID of some app that you register on the backend?

Reply:
@zmb3 What do you suggest for this? I'm thinking we just merge this PR so at least the docs match current behavior, then make a follow up PR that either makes that field optional or removes it and updates the docs to reflect that.

Reply:
@zmb3 Are you ok with the plan in the previous comment?